.github/workflows/website_webhook.yml

```diff
@@ -1,67 +1,42 @@
-name: auto-upload announcements to website
+name: Auto-upload Distro announcements to Bazaar
 
 on:
-  issues:
+  discussion:
     types: [pinned]    
 
 jobs:
  sendAnnouncement:
   runs-on: ubuntu-latest
   steps:
-
-  - name: get release/announcement
-    id: rel_annou
-    env:
-      ISSUE_CONTEXT: ${{ toJson(github.event.issue) }}
+  
+  - name: Fetch announcement
+    id: announcement
     run: |
-      echo $ISSUE_CONTEXT > temp_title.txt
-      echo $(cat temp_title.txt)
-      sed -nr 's/.*"title": (.*), "updated.*/\1/p' <temp_title.txt >title.txt
-      echo $(cat title.txt)
-      if [[ $(cat title.txt) == *"Release"* ]]
+      category="${{ github.event.discussion.category.name }}"
+      title="${{ github.event.discussion.title }}"
+      body="${{ github.event.discussion.body }}"
+      
+      echo "::set-output name=title::${title}"
+      echo "::set-output name=summary::${body:0:500}"
```

Comment:
We can pass only the body, and do the parsing for the summary on the website side (easy change there, and we can remove one output)
```suggestion
```

Reply:
The proposed commit would overwrite too many lines! Can you amend it?
Also, we need to update the dispatching part.